### DISCUSSION TITLE
The email with the github loss

### DISCUSSION THREAD

**Discussion by robinzixuan** [community/community#38650] [Other Feature Feedback, Questions, & Ideas]
The email with my GitHub was lost, and I cannot log in again. What should I do?

**Comment by radar-blip** [comment]
For account related issues, see here: [Support page](https://support.github.com/contact/account)

**Reply by DSDmark** [reply]
HI @robinzixuan,

if you have your **token,** then add with your email when you write to **get support**. It's help you to recover your account.

**Reply by queenofcorgis** [reply]
Hi there - @robinzixuan I'd caution against sharing any sensitive information via email to support unless they ask to keep yourself safe 😄
